elasticsearch/src/main/java/org/hibernate/search/elasticsearch/logging/impl/Log.java

```diff
@@ -189,4 +190,45 @@
 	)
 	void unsupportedDynamicBoost(Class<?> boostStrategyType, Class<?> entityType, String fieldPath);
 
+	@Message(id = ES_BACKEND_MESSAGES_START_ID + 33,
+			value = "Validation failed for schema of index '%1$s'"
+	)
+	SearchException schemaValidationFailed(String indexName, @Cause Exception cause);
+
+	@Message(id = ES_BACKEND_MESSAGES_START_ID + 34,
+			value = "No mappings available from Elasticsearch for validation of index '%1$s'"
```

Comment:
When would this happen? I assume it might mean that the index was not defined; could we suggest a possible cause in the error?